tests/unit/linalg/test_ode2.cpp

```diff
@@ -112,18 +125,49 @@ TEST_CASE("Second order ODE methods",
          std::cout<<std::setw(12)<<err_u[0]
                   <<std::setw(12)<<err_du[0]<<std::endl;
 
+         Array<Vector> uh(ode_solver->GetMaxStateSize());
```

Comment:
The `mfem::Array` class only supports POD types, so this line should use something like `std::vector<Vector>` instead.

Reply:
I had to google POD. But I guess it stands for plain-old-data, and assumes all the data is contigous in memory, or something like that, such that memcpy (etc) works... Just to make sure I understand and in an attempt to not make the mistake in the future.

Reply:
Made the change.

Reply:
Here is a precise definition: https://en.cppreference.com/w/cpp/named_req/PODType.

The main reason for the POD requirement in `Array` is that we do not call contructors or destructors for the array entries.